helper/src/main/java/me/lucko/helper/item/ItemStackBuilder.java

```diff
@@ -74,7 +74,7 @@ public static ItemStackBuilder of(ConfigurationSection config) {
         return ItemStackReader.DEFAULT.read(config);
     }
 
-    private ItemStackBuilder(ItemStack itemStack) {
+    public ItemStackBuilder(ItemStack itemStack) {
```

Comment:
But why? It doesn't hurt anything, but it doesn't help anything, either. What is the benefit of making this change?

Reply:
@Stijn-van-Nieulande reported on discord that they had to use a dirty hack (using reflection to access the private constructor) to not have attributes hidden by default.

Reply:
@jeuxjeux20 A good example of why an explanation as to the reasoning would have been helpful.

Still, seems irrelevant when this PR also removes `#hideAttributes` from the static accessor.

Reply:
@FerusGrim First of all... The details for this pull request are in discord and this pull request was at the request of lucko himself.
And I didn't remove #hideAttributes from the static accessor, I only made a reference to the other already existing method to create a itemstack

Reply:
>The details for this pull request are in discord and this pull request was at the request of lucko himself.

So it should be easy to copy it into the PR?

I'm not part of the discord, so that doesn't help me (or anyone else who isn't part of the Discord).

>And I didn't remove #hideAttributes from the static accessor, I only made a reference to the other already existing method to create a itemstack

Ah, yea, I wasn't in full file view, so I didn't notice that it just routes it to another method which has it.

Reply:
> I'm not part of the discord, so that doesn't help me (or anyone else who isn't part of the Discord).

I know that can be annoying to have stuff not centralized on one platform...
...but here is the invite link if you want to join it :D https://discord.gg/Rnbpc7s